src/Uno.UI/ts/Clipboard.ts

```diff
@@ -50,12 +52,17 @@ namespace Uno.Utils {
 			return "ok";
 		}
 
-		public static getText(): Promise<string> {			
+		public static getText(): Promise<string> {
+			// we return "" on failure instead of null to avoid crashing with an NRE if there
+			// are no try blocks in the stack frames above.
 			const nav = navigator as NavigatorClipboard;
 			if (nav.clipboard) {
-				return nav.clipboard.readText();				
+				return nav.clipboard.readText().catch(reason => {
+					console.error(`Failed to read from clipboard: ${reason}`);
+					return "";
+				});
 			}
-			return Promise.resolve(null)
+			return Promise.resolve("")
```

Comment:
Should we raise something on the managed side? Maybe the WinRT API has some sort of reporting mecanism?

Reply:
We do raise on the managed side, but this is bad because it crashes the app. The WinRT API doesn't mention anything (mainly because there is no clipboard access permission setting in Windows), so without a try-catch somewhere, we end up crashing. This is unrelated to the original problem reported in https://github.com/unoplatform/uno/issues/19365, where it was crashing because the `readText` promise was throwing and we needed to add a `catch`.